src/Layer/Layer.js

```diff
@@ -239,8 +239,16 @@ class Layer extends THREE.EventDispatcher {
      * @param {boolean} [clearCache=false] Whether to clear the layer cache or not
      */
     // eslint-disable-next-line
-    delete(clearCache) {
-        console.warn('Function delete doesn\'t exist for this layer');
+    dispose(clearCache) {
+        if (clearCache) {
```

Comment:
Thanks! Now that you have factorized it in the Layer, you can remove this part from the other layers `dispose` methods and call `super.dispose();` instead.

Reply:
Alright, gotcha. Hopefully that fixes it.